tests/integration_test/transcoding_options_test.go

```diff
@@ -73,9 +73,9 @@ func TestTranscodingPrintOptions(t *testing.T) {
 			clientProtocol:                     "http",
 			httpMethod:                         "POST",
 			method:                             "/v1/shelves/100/books?key=api-key",
-			bodyBytes:                          []byte(`{"id": 4, "price_in_usd": 100}`),
+			bodyBytes:                          []byte(`{"id": 4, "priceInUsd": 100}`),
```

Comment:
Can the request message also be in snake_case? I assume so, just want to double-confirm

Reply:
Yes,  the old code was using snake_case in the request JSON, it works.  This flag only control transcoding respond from proto to JSON.